rpc/grpc/server/services/blockservice/service.go

```diff
@@ -0,0 +1,129 @@
+package blockservice
```

Comment:
Perhaps this could be renamed to `block`?
I'm not a fan of repetitive naming conventions, and I'd like the package for the
pruning service be named simply `services/pruning`.

Reply:
Then again, there's already `versionservice`.

Reply:
The reason for the naming convention here is to facilitate human-readable construction like:

```golang
svr := blockservice.New(store, eventBus, logger)
```

If we rename the package to `block` that would become:

```golang
svr := block.New(store, eventBus, logger)
```

which doesn't make sense. I can't think of a constructor name off-hand right now that would make sense if we named the package `block`.